src/test/run-pass/issue-48070.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Rust Project Developers. See the COPYRIGHT
```

Comment:
can you rename this file to `run-pass/nll/issue-48070.rs`?

Or, better yet, `ui/nll/issue-48070.rs`? (In that case, you would add a `// run-pass` comment in the header)